hardhat.config.ts

```diff
@@ -120,7 +120,7 @@ const config: HardhatUserConfig = {
       url: "https://goerli-rollup.arbitrum.io/rpc",
       saveDeployments: true,
       accounts: { mnemonic },
-      companionNetworks: { l1: "goerli" },
+      companionNetworks: { l1: "sepolia" },
```

Comment:
Are you sure you want to commit this change? I don't think `arbitrum-goerli` can actually communicate with `sepolia`, only `goerli` system contracts.
```suggestion
      companionNetworks: { l1: "goerli" },
```

Reply:
Yeah - this can remain as goerli. Communication obviously isn't possible, but I didn't want anyone to accidentally refer to the wrong HubPool instance when looking at the new deployments on Etherscan.

Reply:
ah yeah but i would rather keep this committed as linked to `goerli`